src/garage/tf/algos/rl2.py

```diff
@@ -360,6 +356,19 @@ def spec(self):
         """
         return self._spec
 
+    def _pad_zeros(self, obs):
```

Comment:
does this not exist in `g.misc.tensor_utils`?

Reply:
https://github.com/rlworkgroup/garage/blob/master/src/garage/misc/tensor_utils.py#L91

Reply:
fixed